VideoLocker/src/main/java/org/edx/mobile/authentication/LoginAPI.java

```diff
@@ -161,38 +166,38 @@ public AuthResponse registerUsingFacebook(@NonNull Bundle parameters, @NonNull S
     }
 
     @NonNull
-    public ResetPasswordResponse resetPassword(@NonNull String email) throws HttpException {
-        return loginService.resetPassword(email);
-    }
-
-    @NonNull
     private void register(Bundle parameters) throws Exception {
```

Comment:
So, as per my [reply][1] on a similar comment below, it would actually be safest to do a generic catch here, and as the specific exceptions aren't being caught anywhere, I'm leaving it as it is for now.

[1]: https://github.com/edx/edx-app-android/pull/799#discussion_r77169177